config/config.go

```diff
@@ -270,6 +270,10 @@ type Config struct {
 
 	LuaModules *listFlag `yaml:"lua-modules"`
 	LuaSources *listFlag `yaml:"lua-sources"`
+
+	EnableOpenPolicyAgent         bool   `yaml:"enable-open-policy-agent"`
+	OpenPolicyAgentConfigTemplate string `yaml:"open-policy-agent-config-template"`
+	OpenPolicyAgentEnvoyMetadata  string `yaml:"open-policy-agent-envoy-metadata"`
```

Comment:
I am not sure if we should leak implementation details (`envoy`) here

Reply:
Your call, we can strip `envoy`. the format however is specific and needs to follow the structure of https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/base.proto#config-core-v3-metadata